engine/src/main/kotlin/io/provenance/scope/definition/DefinitionService.kt

```diff
@@ -7,6 +7,7 @@ import io.provenance.scope.contract.proto.Commons.DefinitionSpec
 import io.provenance.scope.encryption.model.KeyRef
 import io.provenance.scope.objectstore.client.CachedOsClient
 import io.provenance.scope.objectstore.util.base64Decode
+import io.provenance.scope.util.forThread
```

Comment:
unused?

Reply:
used, just moved from MemoryClassLoader to an extension function